ic-stable-structures/Cargo.toml

```diff
@@ -5,6 +5,8 @@ edition.workspace = true
 
 [dependencies]
 dfinity-stable-structures = { workspace = true }
+ic-cdk = { workspace = true }
```

Comment:
But I need it to get the `ic_cdk::api::stable::WASM_PAGE_SIZE_IN_BYTES` to implement `Memory` for `MemoryMappedFileMemory`. I can only make this dependency optional

Reply:
Because the `MemoryManager` relies on that value when allocating new memory. If it calls `memory.grow(1)` it expects that exactly `WASM_PAGE_SIZE_IN_BYTES` bytes is allocated